### PR TITLE
codegen: arrays can no longer be relationships

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

File: generator/processor.go [hunk 1]
```diff
@@ -248,6 +248,11 @@ func (p *Processor) processFields(s *types.Struct, done []*types.Struct, root bo
 		}
 
 		p.processField(field, f.Type(), done, root)
+		if field.Kind == Invalid {
+			fmt.Printf("WARNING: arrays of relationships are not supported. Field %s will be ignored.\n", field.Name)
```
Comment (dpordomingo) [inline generator/processor.go]: the preffix shouldn't be `kallax:`?

Comment (dpordomingo) [inline generator/processor.go]: the preffix shouldn't be `kallax:`?

File: generator/processor.go [hunk 2]
```diff
@@ -348,7 +353,7 @@ func (p *Processor) processField(field *Field, typ types.Type, done []*types.Str
 			field.SetFields(subfs)
 		}
 	default:
-		fmt.Printf("Ignored field %s of type %s\n", field.Name, field.Type)
+		fmt.Printf("WARNING: Ignored field %s of type %s\n", field.Name, field.Type)
```
Comment (dpordomingo) [inline generator/processor.go]: the preffix shouldn't be `kallax:`?

Comment (erizocosmico) [inline generator/processor.go]: Nope, it's not an `error`, it's a warning message printed to the user.

File: generator/processor.go (hunk 1 above)
Comment (erizocosmico) [inline generator/processor.go]: same

**Pull request merged**